### PR TITLE
updated SCIM images

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated SCIM images and attributes step

### CODE REVIEW

File: docker-hub/scim.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ Before making SCIM configuration changes in your IdP, navigate to [Docker Hub](h
 
 ![scim-app-provisioning](images/scim-app-provisioning.png){:width="700px"}
 
+6. Remove the unwanted attributes from your mappings and fields from the Profile editor.
```
Comment (craig-osterhout) [inline docker-hub/scim.md:43]: In the preview, this is displaying as step 1 instead of 6.

Comment (ivan-californias) [inline docker-hub/scim.md:43]: Profile editor is used to modify custom user attributes, we don't need them to do that, they just need to remove them from the Attributes mappings
```suggestion
6. Remove all fields that are not supported from your Attributes Mappings.
```

And add this image a
Untitled
s reference of the attributes we do support:

Comment (jerae-duffin) [reply docker-hub/scim.md]: Thank you, updated!

Comment (jerae-duffin) [reply docker-hub/scim.md]: Thank you, updated!

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Review by ivan-californias — APPROVED** [review]
👍  Thank you

**Pull request merged**